jbmc/src/java_bytecode/remove_instanceof.cpp

```diff
@@ -122,8 +131,6 @@ std::size_t remove_instanceoft::lower_instanceof(
   // disjunction of all possible object types. According to the Java
   // specification, null instanceof T is false for all possible values of T.
   // (http://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.20.2)
```

Comment:
maybe makes sense to adapt this comment now

Reply:
Disagree-- the code now generated has the form
```
clsid = ptr == null ? "" : ptr->@class_identifier
isA = ptr != null && clsid == "A"
```

The comment applies to the second line, and is placed prior to the second line as before. This change simply avoids deref'ing a pointer that can still be null.

Reply:
ok then